src/views/v2/commerce/checkout/cart.php

```diff
@@ -35,7 +36,7 @@
 
 	<?php $this->template( 'checkout/cart/items', [ 'post' => $post ] ); ?>
 
-	<?php $this->template( 'checkout/cart/billing_identification' ); ?>
+	<?php $this->template( 'checkout/cart/billing-details', [ 'user' => $user ] ); ?>
```

Comment:
The billing details shouldn't be part of the cart, we should move this outside of it. This comment is a reminder for the FE audit. Shouldn't be addressed now.

Reply:
Will add a `@todo` for this.